Library/Formula/dmd.rb

```diff
@@ -65,6 +57,27 @@ def install
     end
   end
 
+  # Fix up dmd.conf to point to the correct include paths.
+  def fix_dmd_conf
+    # Homebrew's etc.install method doesn't let you overwrite etc files,
+    # so edit it in place if it's there:
+    conf = etc/"dmd.conf"
+    if conf.exist?
+      inreplace conf, /^DFLAGS=.+$/, "DFLAGS=-I#{include}/d2 -L-L#{lib}"
+    else
+      conf.write <<-EOS.undent
+        [Environment]
+        DFLAGS=-I#{include}/d2 -L-L#{lib}
+        EOS
```

Comment:
Ooooh.   I think I figured it out.  /usr/local/opt/dmd is a symlink to the current version of dmd, so should always be correct.  Handy!